doc/classes/Dictionary.xml

```diff
@@ -9,36 +9,82 @@
 		Erasing elements while iterating over them [b]is not supported[/b] and will result in undefined behavior.
 		[b]Note:[/b] Dictionaries are always passed by reference. To get a copy of a dictionary which can be modified independently of the original dictionary, use [method duplicate].
 		Creating a dictionary:
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var my_dir = {} # Creates an empty dictionary.
 		var points_dir = {"White": 50, "Yellow": 75, "Orange": 100}
 		var another_dir = {
 		    key1: value1,
 		    key2: value2,
 		    key3: value3,
 		}
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		var myDir = new Godot.Collections.Dictionary(); // Creates an empty dictionary.
+		var pointsDir = new Godot.Collections.Dictionary
+		{
+		    {"White", 50},
+		    {"Yellow", 75},
+		    {"Orange", 100}
+		};
+		[/csharp]
+		[/codeblocks]
 		You can access a dictionary's values by referencing the appropriate key. In the above example, [code]points_dir["White"][/code] will return [code]50[/code]. You can also write [code]points_dir.White[/code], which is equivalent. However, you'll have to use the bracket syntax if the key you're accessing the dictionary with isn't a fixed string (such as a number or variable).
-		[codeblock]
-		export(String, "White", "Yellow", "Orange") var my_color
+		[codeblocks]
+		[gdscript]
+		export(string, "White", "Yellow", "Orange") var my_color
 		var points_dir = {"White": 50, "Yellow": 75, "Orange": 100}
-
 		func _ready():
 		    # We can't use dot syntax here as `my_color` is a variable.
 		    var points = points_dir[my_color]
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		[Export(PropertyHint.Enum, "White,Yellow,Orange")]
+		string MyColor;
+		var pointsDir = new Godot.Collections.Dictionary
```

Comment:
```suggestion
		public string MyColor { get; set; }
		public Godot.Collections.Dictionary pointsDir = new Godot.Collections.Dictionary
```

`MyColor` should be a property since it has special behavior in the editor (enum) and we want to leave room for also having special behavior through C# code without breaking ABI compatibility. One could argue that this adds complexity to the documentation, but I think it's a good idea to showcase best practices.

> `The contextual keyword 'var' may only appear within a local variable declaration or in script code`

Also, use explicit access modifiers as per the style guide.